src/handler/http/models/dashboards.rs

```diff
@@ -0,0 +1,280 @@
+    }
+}
+
+impl ListDashboardsQuery {
+    pub fn into(self, org_id: &str) -> config::meta::dashboards::ListDashboardsParams {
+        match self {
+            Self {
+                folder: Some(f),
+                title: Some(t),
+            } => config::meta::dashboards::ListDashboardsParams::new(org_id)
+                .with_folder_id(&f)
+                .where_title_contains(&t),
+            Self {
+                folder: None,
+                title: Some(t),
+            } => {
+                config::meta::dashboards::ListDashboardsParams::new(org_id).where_title_contains(&t)
+            }
+            Self {
+                folder: Some(f),
+                title: None,
+            } => config::meta::dashboards::ListDashboardsParams::new(org_id).with_folder_id(&f),
+            Self {
+                folder: None,
+                title: None,
+            } => {
+                // To preserve backwards-compatability when no filter parameters
+                // are given we will list the contents of the default folder.
+                config::meta::dashboards::ListDashboardsParams::new(org_id)
+                    .with_folder_id(config::meta::folder::DEFAULT_FOLDER)
+            }
+        }
+    }
+}
+
+impl TryFrom<Vec<config::meta::dashboards::Dashboard>> for ListDashboardsResponseBody {
+    type Error = serde_json::Error;
+
+    fn try_from(value: Vec<config::meta::dashboards::Dashboard>) -> Result<Self, Self::Error> {
+        let dashboards: Result<Vec<ListDashboardsResponseBodyItem>, _> = value
+            .into_iter()
+            .map(TryInto::<ListDashboardsResponseBodyItem>::try_into)
+            .collect();
+        Ok(Self {
+            dashboards: dashboards?,
+        })
+    }
+}
+
+impl TryFrom<config::meta::dashboards::Dashboard> for ListDashboardsResponseBodyItem {
+    type Error = serde_json::Error;
+
+    #[allow(deprecated)]
+    fn try_from(value: config::meta::dashboards::Dashboard) -> Result<Self, Self::Error> {
+        Ok(Self {
+            dashboard_id: value.dashboard_id().unwrap_or_default().to_owned(),
+            title: value.title().unwrap_or_default().to_owned(),
+            description: value.description().unwrap_or_default().to_owned(),
+            role: value.role().unwrap_or_default().to_owned(),
+            owner: value.owner().unwrap_or_default().to_owned(),
+            created: value.created_at_deprecated().unwrap_or_default(),
+            hash: value.hash,
+            version: value.version,
+            // Populate deprecated fields until they are removed from the API.
+            v1: value.v1.as_ref().map(serde_json::to_value).transpose()?,
+            v2: value.v2.as_ref().map(serde_json::to_value).transpose()?,
+            v3: value.v3.as_ref().map(serde_json::to_value).transpose()?,
+            v4: value.v4.as_ref().map(serde_json::to_value).transpose()?,
+            v5: value.v5.as_ref().map(serde_json::to_value).transpose()?,
+        })
+    }
```

Comment:
We need to allow deprecated since we will continue populating the deprecated `v1`...`v5` fields until they are removed as part of #5196